styles/variables.less

```diff
@@ -0,0 +1,4 @@
+@width100:  100%;
+@width0:    0;
+@height100: 100%;
+@height0:   0;
```

Comment:
I'm not sure these are worth having. Honestly, while reviewing I thought @width100 would be 100px so at minimum it is a bit confusing.